src/Sulu/Bundle/HttpCacheBundle/Resources/config/structure-cache-handlers.xml

```diff
@@ -7,6 +7,7 @@
         <parameter key="sulu_http_cache.handler.aggregate.class">Sulu\Component\HttpCache\Handler\AggregateHandler</parameter>
         <parameter key="sulu_http_cache.handler.tags.class">Sulu\Component\HttpCache\Handler\TagsHandler</parameter>
         <parameter key="sulu_http_cache.handler.paths.class">Sulu\Component\HttpCache\Handler\PathsHandler</parameter>
+        <parameter key="sulu_http_cache.handler.url.class">Sulu\Component\HttpCache\Handler\UrlHandler</parameter>
```

Comment:
We actually don't add the parameters like this anymore, because this is deprecated in Symfony3. Please just use the content of this parameter in the `class` attribute directly.